backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  name        String?
+  lastRunDate DateTime? @map("last_run_date") @db.Timestamptz(6)
+  createdAt   DateTime  @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt   DateTime  @updatedAt @map("updated_at") @db.Timestamp(6)
+
+  @@map("cron_job")
+}
+
+// Note: [ethnicity, gender, sexual_orientation, how_did_you_hear] formerly max length 64
+// Note: [how_did_you_hear, race] formerly max array length 64
+model Demographics {
+  id                String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt         DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt         DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  ethnicity         String?
+  gender            String?
+  sexualOrientation String?       @map("sexual_orientation")
+  howDidYouHear     String[]      @map("how_did_you_hear")
+  race              String[]
+  applications      Applications?
+
+  @@map("demographics")
+}
+
+model GeneratedListingTranslations {
+  id             String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt      DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt      DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  listingId      String        @map("listing_id") @db.VarChar
+  jurisdictionId String        @map("jurisdiction_id") @db.VarChar
+  language       LanguagesEnum
+  translations   Json
+  timestamp      DateTime      @db.Timestamp(6)
+
+  @@map("generated_listing_translations")
+}
+
+// Note: [birth_month, birth_day, birth_year] formerly max length 8; 
+//       [phone_number, phone_number_type] formerly max length 16
+//       [first_name, middle_name, last_name, relationship] formerly max length 64;
+// Note: [email_address] formerly enforced lower case
+model HouseholdMember {
+  id                         String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                  DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                  DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  orderId                    Int?          @map("order_id")
+  firstName                  String?       @map("first_name")
+  middleName                 String?       @map("middle_name")
+  lastName                   String?       @map("last_name")
+  birthMonth                 String?       @map("birth_month")
+  birthDay                   String?       @map("birth_day")
+  birthYear                  String?       @map("birth_year")
+  emailAddress               String?       @map("email_address")
+  phoneNumber                String?       @map("phone_number")
+  phoneNumberType            String?       @map("phone_number_type")
+  noPhone                    Boolean?      @map("no_phone")
+  sameAddress                YesNoEnum     @map("same_address")
```

Comment:
Should this + the similar one below be a boolean?

Reply:
this is a similar case to something I wrote about above, we basically created an enum but the hard way previously, this makes explicit (and in less code) that this is an enum of Yes or No